src/langcheck/augment/en/_jailbreak_template.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from .._common._jailbreak_template import _jailbreak_template
+
+AVAILABLE_JAILBREAK_TEMPLATES = [
+    "basic",
+    "dan",
+    "good_vs_evil",
+    "john",
+    "universal_adversarial_suffix",
+]
+
+
+def jailbreak_template(
+    instances: list[str] | str,
+    templates: list[str] | None = None,
+    *,
+    num_perturbations: int = 1,
+) -> list[str]:
+    """Applies jailbreak templates to each string in instances.
+
+    Args:
+        instances: A single string or a list of strings to be augmented.
+        templates: A list of applied templates. If None, some templates are
```

Comment:
Maybe add the list of options here? That way, the options will show up for users in vscode

Reply:
Added